### ISSUE TITLE
GameGuru MAX - Not Sure SSAO Is Working Correctly?

### ISSUE THREAD

**Issue by MonkeyFrogStudio** [TheGameCreators/GameGuruRepo#4153]
Here's part of a scene with SSAO turned off:

![ssao_1](https://user-images.githubusercontent.com/36735507/217060700-fdbce16b-159e-4399-817d-1dd61298691e.png)

And this is the same scene with SSAO enabled, but set very low to 0.1:

![ssao_2](https://user-images.githubusercontent.com/36735507/217060773-8f470a94-88bd-417e-bbf3-4e9e5a3e5d7d.png)

Now let's turn this up to 1.0:

![ssao_3](https://user-images.githubusercontent.com/36735507/217060875-72473a18-9eb9-4235-b5af-65a944227b50.png)

And then to something way too strong, like 5.0:

![ssao_4](https://user-images.githubusercontent.com/36735507/217060941-29a3e1bb-828f-41c0-befb-c41d87aafd65.png)

Instead of the cracks, cavities, and crevices getting darker and darker as SSAO is increased, the entire image or screen looks to be getting darker and darker.

**Comment by OldFlak** [comment]
Yes, seeing the same here. It used to work fine but in last few builds it just makes the whole scene darker or lighter.
